shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -32,6 +32,32 @@ namespace flutter {
 
 namespace {
 
+// Update the thread priority for the Windows engine.
+static void WindowsPlatformThreadPrioritySetter(
+    FlutterThreadPriority priority) {
+  // TODO(99502): Add support for tracing to the windows embedding so we can
+  // mark thread priorities and success/failure.
+  switch (priority) {
+    case FlutterThreadPriority::kBackground: {
+      SetPriorityClass(GetCurrentProcess(), BELOW_NORMAL_PRIORITY_CLASS);
```

Comment:
Shouldn't this be SetThreadPriority/GetCurrentThread?

Reply:
Yeah, just realized.

Reply:
Done